packages/react-app/src/modules/Counter/CounterDetail/CounterDetail.component.js

```diff
@@ -35,13 +35,20 @@ const Component = ({ onIncrement, onDecrement, counterId, count, label }) => {
       <main className="CounterDetail__main">
         {or(isNil(counterId), isNil(count)) ? noSelectedCounterView : selectedCounterView}
       </main>
+      <footer>
+        <button
+          onClick={onNavigateToOverview}
+          type="button"
+        >Overview</button>
```

Comment:
can we call this "Back" or "Back to Counters"

I had to rack my brain to firgure out what overiew was :D